client/src/css/partials/_imaging.scss

```diff
@@ -220,3 +220,9 @@ div.img_history {
 input[name=gap] {
 	width: 30px;
 }
+
```

Comment:
I think we should move it here to be more specific to the imaging
I think we should centre the text
I think we should use the content-box model so it correctly calculates the sizes